examples/kubernetes/configure.sh

```diff
@@ -3,6 +3,13 @@
 # This script generates config.sh, which is a site-local config file that is not
 # checked into source control.
 
+# Erase any existing file since we append from here on.
+> config.sh
+
+# Custom Docker image.
+read -p "Vitess Docker image (leave empty for default) []: "
```

Comment:
I would prefer if we don't prompt on this by default. Too many choices for users. The default will be good enough for 99% of them.

Can you add a flag for it instead?